ispdb/inbox.lv

```diff
@@ -2,25 +2,49 @@
 <clientConfig version="1.1">
   <emailProvider id="inbox.lv">
     <domain>inbox.lv</domain>
-    <!-- Same as inbox.lt TODO merge? -->
-    <displayName>Inbox.lv</displayName>
-    <displayShortName>Inbox.lv</displayShortName>
+    <domain>inbox.eu</domain>
+    <domain>inbox.lt</domain>
+    <domain>inbox.la</domain>
+    <displayName>%EMAILDOMAIN%</displayName>
+    <displayShortName>%EMAILDOMAIN%</displayShortName>
+    <incomingServer type="imap">
+      <hostname>mail.%EMAILDOMAIN%</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <authentication>password-cleartext</authentication>
+      <username>%EMAILADDRESS%</username>
+    </incomingServer>
     <incomingServer type="pop3">
-      <hostname>mail.inbox.lv</hostname>
+      <hostname>mail.%EMAILDOMAIN%</hostname>
       <port>995</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
+      <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
       <pop3>
         <leaveMessagesOnServer>true</leaveMessagesOnServer>
       </pop3>
     </incomingServer>
     <outgoingServer type="smtp">
-      <hostname>mail.inbox.lv</hostname>
+      <hostname>mail.%EMAILDOMAIN%</hostname>
       <port>587</port>
       <socketType>STARTTLS</socketType>
-      <username>%EMAILLOCALPART%</username>
+      <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
     </outgoingServer>
+    <documentation url="https://help.%EMAILDOMAIN%/?keyword=email_clients_settings">
+       <descr lang="en">Configure Thunderbird 2.0 for IMAP/POP3/SMTP</descr>
+       <descr lang="lv">E-pasta klientu konfigurācija (IMAP/POP3/SMTP)</descr>
+       <descr lang="lt">Pašto programų nustatymai (IMAP, POP3, SMTP)</descr>
+       <descr lang="ru">Настройки для почтовых программ (IMAP, POP3,SMTP и др.)</descr>
+    </documentation>
   </emailProvider>
```

Comment:
Please add:
```
<enable visiturl="https://help.inbox.eu/help/question/443/37">
  <instruction>Enable external POP3/IMAP and SMTP access in your account settings</instruction>
  <instruction lang="lv">...</instruction>
</enable>
```
and add translations for the main languages of this ISP.

Reply:
Same for the other 2 config files.

Reply:
Note that no placeholders are allowed in the URL. However, it would be OK to use inbox.eu, for example, for all the ISPs. Don't worry too much about the URL, the page is not currently used, but we may add a feature to show this page to the user. At the least, it captures the information for us here in the file, in a structured form.